### PR TITLE
v21.1.13 release notes

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://cockroachlabs.atlassian.net/browse/DOC-1882.

### CODE REVIEW

File: releases/v21.1.13.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+- Fixed a bug causing CockroachDB to return a spurious "context canceled" error for a query that actually succeeded in extremely rare cases. [#73959][#73959]
+- Fixed a bug which caused corruption of [partial indexes](../v21.2/partial-indexes.html), which could cause incorrect query results. The bug was only present when two or more partial indexes in the same table had identical `WHERE` clauses. This bug had been present since version 21.1.0. [#74475][#74475]
+
+### Performance improvements
+
+- Improved [`IMPORT INTO`](../v21.1/import.html) performance in cases where it encounters large numbers of unresolved write intents. [#72272][#72272]
+- The performance of transaction deadlock detection is now more stable even with significant transaction contention. [#71743][#71743]
+- Bulk ingestion of small write batches (e.g., index backfill into a large number of ranges) is now throttled, to avoid buildup of read amplification and associated performance degradation. Concurrency is controlled by the new cluster setting `kv.bulk_io_write.concurrent_addsstable_as_writes_requests`. [#74072][#74072]
+
+### Doc updates
```
Comment (nickvigilante) [inline releases/v21.1.13.md:91]: @ericharmeling If there are no doc updates, for example, should we not remove the section altogether?

**Comment by nickvigilante** [comment]
@ericharmeling Other than the empty Doc updates section, everything LGTM!

File: releases/v21.1.13.md (hunk 1 above)
Comment (ericharmeling) [reply releases/v21.1.13.md]: Thank you. Yes.

Done.

**Review by ericharmeling — COMMENTED** [review]
TFTR !

**Review by nickvigilante — APPROVED** [review]

**Pull request merged**